### DISCUSSION TITLE
micropython, micro:bit, and DFRobot Maqueen Plus V2 together at last.

### DISCUSSION THREAD

**Discussion by jdonwells** [micropython/micropython#13176] [nRF]
https://github.com/jdonwells/micropython-MaqueenPlusV2

The file maqueenplusv2.py has all the functions you need to control a Maqueen Plus V2 with micropython on a BBC micro:bit.

**Comment by lebalz** [comment]
This one aswell 😉
https://gbsl-informatik.github.io/maqueen-plus-v2-mpy/docs/

**Reply by jdonwells** [reply]
Excellent. I assume it is set up for version 2.0 Maqueen. In version 2.1 Maqueen the line sensor address' are in the opposite order! Add a function to init the Maqueen based on the version.
